raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -248,27 +248,27 @@ contract MonitoringService is Utils {
         require(service_registry.hasValidRegistration(monitoring_service_address), "service not registered");
 
         TokenNetwork.ChannelState channel_state;
-        uint256 settle_block_number;
-        (settle_block_number, channel_state) = token_network.getChannelInfo(
+        uint256 settle_window;
+        (settle_window, channel_state) = token_network.getChannelInfo(
             channel_identifier, closing_participant, non_closing_participant
         );
         require(channel_state == TokenNetwork.ChannelState.Closed, "channel not closed");
 
         uint256 settle_timeout = token_network.settle_timeout();
-        require(settle_block_number >= settle_timeout, "too low settle block number");
-        uint256 assumed_close_block = settle_block_number - settle_timeout;
+        require(settle_window >= settle_timeout, "too low settle block number");
+        uint256 assumed_close_timestamp = settle_window - settle_timeout;
```

Comment:
This is not assumed anymore after https://github.com/raiden-network/raiden-contracts/pull/1557 , right?